### PR TITLE
feat: added SignIn page

### PR DESCRIPTION
**Pull request by jDineshKarthic**
Contains code for my login page.

### CODE REVIEW

File: authentication/loginPage/login.css [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+    font-family: 'Arima Madurai', cursive;
+
+}
+
+header {
+    z-index: 5;
+    padding: 1rem;
+    top: 0;
+    position: sticky;
+    top: 0;
```
Comment (SaurabhChawda) [inline authentication/loginPage/login.css]: Remove repeat properties.

File: authentication/loginPage/login.css [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+    width: 100%;
+    background-color: var(--dk-cream);
+}
+
+.flex-sa {
+    display: flex;
+    flex-direction: row;
```
Comment (SaurabhChawda) [inline authentication/loginPage/login.css]: no need to use flex-direction: row.

File: authentication/loginPage/login.css [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+}
+
+.arima_madurai_100 {
+    font-family: 'Arima Madurai', cursive;
+    font-weight: 100;
+}
+
+.input-items {
+    padding: 0.6rem;
+    padding-right: 4rem;
```
Comment (SaurabhChawda) [inline authentication/loginPage/login.css:51]: you can use like that (padding: 0.6rem 4rem 0.6rem 0rem)

File: authentication/loginPage/login.css [hunk 4]
```diff
@@ -0,0 +1,267 @@
...
+
+a {
+    text-decoration: none;
+
+}
+
+ul {
+
+    display: flex;
+    flex-direction: row;
```
Comment (SaurabhChawda) [inline authentication/loginPage/login.css:72]: Here also

File: authentication/loginPage/login.css [hunk 5]
```diff
@@ -0,0 +1,267 @@
...
+    background-color: var(--lt-pink);
+}
+
+.form-signin {
+
+    /* display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-start; */
+    /* background-color: var(--lt-choco) */
```
Comment (SaurabhChawda) [inline authentication/loginPage/login.css]: remove if no need

File: authentication/loginPage/login.css [hunk 6]
```diff
@@ -0,0 +1,267 @@
...
+.email {
+    color: var(--lt-maroon);
+    margin-bottom: 0.5rem;
+}
+
+.email-text {
+
+    background: transparent;
+    border: none;
+    border-bottom: 2px solid #dcd9d9;
```
Comment (SaurabhChawda) [inline authentication/loginPage/login.css:204]: if you are using border then why are giving border: none

File: authentication/loginPage/login.css [hunk 7]
```diff
@@ -0,0 +1,267 @@
...
+
+.password {
+    color: var(--lt-maroon);
+    margin-bottom: 0.5rem;
+}
+
+.password-text {
+    background: transparent;
+    border: none;
+    border-bottom: 2px solid var(--lt-grey);
```
Comment (SaurabhChawda) [inline authentication/loginPage/login.css:224]: here also

File: authentication/loginPage/login.html [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+              <button class="login-button">Login</button>
+            </div>
+            <div class="create-new-account">
+              <span class="crete-new-account-text"
+                ><a href=""> Create New Account</a></span
+              >
+            </div>
+          </div>
+        </form>
+      </div> -->
```
Comment (SaurabhChawda) [inline authentication/loginPage/login.html]: remove if no need

**Review by SaurabhChawda — APPROVED** [review]

File: .vscode/settings.json [hunk 9]
```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5503
```
Comment (tanishq20) [inline .vscode/settings.json:2]: Don't run two live servers at the same time. Every time it changes the port.

File: authentication/loginPage/login.css [hunk 10]
```diff
@@ -0,0 +1,267 @@
...
+    font-family: 'Arima Madurai', cursive;
+    font-weight: 100;
+}
+
+.input-items {
+    padding: 0.6rem;
+    padding-right: 4rem;
+    border-radius: 1rem;
+    border: none;
+    /* color: var(--dk-grey); */
```
Comment (tanishq20) [inline authentication/loginPage/login.css:54]: remove cmt if not needed

File: authentication/loginPage/login.css (hunk 2 above)
Comment (jDineshKarthic) [reply authentication/loginPage/login.css]: Thanks for review , i have removed it.

File: authentication/loginPage/login.css (hunk 3 above)
Comment (jDineshKarthic) [reply authentication/loginPage/login.css:51]: ok, i'll make the changes

File: authentication/loginPage/login.html (hunk 8 above)
Comment (jDineshKarthic) [reply authentication/loginPage/login.html]: Yes i have removed it

File: authentication/loginPage/login.css (hunk 10 above)
Comment (jDineshKarthic) [reply authentication/loginPage/login.css:54]: ok i'll remove it

File: .vscode/settings.json (hunk 9 above)
Comment (jDineshKarthic) [reply .vscode/settings.json:2]: thanks for reminding

**Pull request merged**